mex/extractors/blueant/transform.py

```diff
@@ -75,15 +77,15 @@ def transform_blueant_sources_to_extracted_activities(
         # find responsible unit
         department = source.department.replace("(h)", "").strip()
         if department in unit_stable_target_ids_by_synonym:
-            department_id = unit_stable_target_ids_by_synonym.get(department)
+            department_ids = unit_stable_target_ids_by_synonym.get(department, [])
```

Comment:
again, this fallback is not needed

```suggestion
            department_ids = unit_stable_target_ids_by_synonym.get(department)
```